docs/running_the_federation.rst

```diff
@@ -9,13 +9,207 @@ Run the Federation
 
 The following are workflows you can consider when creating a federated learning setup.
 
-:doc:`source/workflow/running_the_federation.agg_based`
-    With this workflow, the federation is terminated when the experiment is finished.
-    
 :doc:`source/workflow/director_based_workflow`
     A federation created with this workflow continues to be available to distribute more experiments in series.
 
-   
+:doc:`source/workflow/running_the_federation.agg_based`
+    With this workflow, the federation is terminated when the experiment is finished.
+
+.. # Copyright (C) 2020-2021 Intel Corporation
+.. # SPDX-License-Identifier: Apache-2.0
+
+.. _director_workflow:
+
+************************
+Director-Based Workflow
+************************
+
+A director-based workflow uses long-lived components in a federation. These components continue to be available to distribute more experiments in the federation.
+
+- The *Director* is the central node of the federation. This component starts an *Aggregator* for each experiment, sends data to connected collaborator nodes, and provides updates on the status.
+- The *Envoy* runs on collaborator nodes connected to the *Director*. When the *Director* starts an experiment, the *Envoy* starts the *Collaborator* to train the global model.
+
+
+The director-based workflow comprises the following roles and their tasks:
+
+    - :ref:`establishing_federation_director`
+    - :ref:`establishing_federation_envoy`
+    - :ref:`establishing_federation_experiment_manager`
+
+Follow the procedure in the director-based workflow to become familiar with the setup required and APIs provided for each role in the federation: *Director manager*, *Collaborator manager*, and *Experiment manager (data scientist)*.
+
+.. note::
+    The Open Federated Learning (|productName|) interactive Python API enables the Experiment manager (data scientists) to define and start a federated learning experiment from a single entry point: a Jupyter\*\  notebook or a Python\*\  script.
+
+    See :doc:`director_based_workflow.interactive_api` for details.
+
+An overview of this workflow is shown below.
+
+.. figure:: ../openfl/static_diagram.svg
+
+.. centered:: Overview of the Director-Based Workflow
+
+
+.. # Copyright (C) 2020-2021 Intel Corporation
+.. # SPDX-License-Identifier: Apache-2.0
+
+
+.. _establishing_federation_director:
+
+Director Manager: Set Up the Director
```

Comment:
Also duplicated section with `openfl/docs/source/workflow/director_based_workflow.roles.rst` file, why not just change this file?

Reply:
Could you please review the content. This duplicate can help you to see the difference between versions. Before merge all of them will be deleted